bundle/manifests/ssp-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,221 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
Did you edit anything in the `bundle` directory? If it only contains generated files, we can add it to `.gitignore` and not commit it.